build/docs/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.0.26] - 2023-03-28
+
+- Update `pacemaker_timeout` from 5 to 10 seconds
```

Comment:
Should this say why it was bumped up? Or was this for comfort?

Reply:
For comfort and making loss less noisy. It was also arbitrarily chosen before as well

Reply:
Updated the comment